subprojects/docs/src/docs/userguide/managing_transitive_dependencies.adoc

```diff
@@ -165,7 +165,66 @@ In the example, the versions of `gson` and `dom4j` are provided by the Spring Bo
 
 Gradle treats all entries in the `<dependencyManagement>` block of a BOM similar to <<#sec:dependency_constraints,Gradle's dependency constraints>>. This means that any version defined in the `<dependencyManagement>` block can impact the dependency resolution result. In order to qualify as a BOM, a `.pom` file needs to have `<packaging>pom</packaging>` set.
 
+However often BOMs are not only providing versions as recommendations, but also a way to override any over version found in the graph. You can enable this behavior by using the `enforcedPlatform` keyword, instead of `platform`, when importing the BOM:
+
+.Importing a BOM, making sure the versions it defines override any other version found in the graph
```

Comment:
"in the graph" may not be necessary here. We probably could make this title a little shorter, but it's not critical that we do so.